product_price_category/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+======================
+Product Price Category
+======================
+
+This module adds a field Price Category on Product Template
+and allow Pricelist to be applied on this field.
+
+Usage
+=====
+
+In Pricelist Form (Sales -> Configuration -> Pricelist), you can add or modify
+an item ("Manage Pricelist Items" access should be checked in the user settings)
+and select "Price Category" in "Applied on".
```

Comment:
"Advanced price rules (discounts, formulas)" pricelist option must be activated to enable this feature, maybe it is better to add this info in the README?

Reply:
@sbejaoui